package.json

```diff
@@ -4,55 +4,58 @@
   "version": "7.2.92",
   "description": "BetterTTV enhances Twitch with new features, emotes, and more.",
   "main": "betterttv.js",
+  "type": "module",
   "scripts": {
-    "start": "node -e 'require(\"./dev/server.js\")()'"
+    "start": "webpack serve --mode development",
+    "build": "webpack --mode development",
+    "build:prod": "webpack --mode production",
+    "lint": "eslint '**/*.js'",
```

Comment:
could we make lint check both eslint and prettier

```
eslint '**/*.js' && prettier --check '**/*.js'
```

Reply:
scripts have been combined